english/bake-cookies/cookies.md

```diff
@@ -10,4 +10,6 @@ Do you:
 
 [Sing a song?](../sing-song/sing.md)
 
-[Make coffee first?](../coffee/coffee.md)
\ No newline at end of file
+[Make coffee first?](../coffee/coffee.md)
+
+[Know why?](../know-why/know-why.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.